easybuild/easyconfigs/r/resolos/resolos-0.3.5.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'resolos'
+version = '0.3.5'
+
+homepage = 'https://gitlab.unige.ch/resolos/resolos'
+description = """Resolos is a toolkit written in Python for maintaining reproducible environments for scientific computations.
+It's main goal is to enable researchers to easily replicate environments through space (running code on HPC environment)
+and time (environment preservation for long term archival).
+For installation and detailed usage, check out the documentation."""
+
+toolchain = {'name': 'system', 'version': ''}
+
+# source_urls = [PYPI_SOURCE]
+# Link copy from pypi.org | PYPI_SOURCE is not working
+sources = ["https://files.pythonhosted.org/packages/15/0b/fa95a349434f77d8afc775dd999152d7612b6baed52083981ac84eaa41e9/resolos-0.3.5.tar.gz"]
```

The parameter `sources` defines the name of the file with the sources, not the full URL. Here you can use the global `SOURCE_TAR_GZ`, which corresponds to `{name}-{version}.tar.gz`: 
```suggestion
sources = [SOURCE_TAR_GZ]
```